src/pybind/mgr/alerts/module.py

```diff
@@ -207,15 +239,31 @@ def _smtp_format_alert(self, code, alert):
     def _send_alert_smtp(self, status, diff):
         # message
         self.log.debug('_send_alert_smtp')
+        if self.emojis:
+            if status['status']=='HEALTH_OK':
+                self.emote=' {}'.format(self.emoji_ok)
+            elif status['status']=='HEALTH_WARN':
+                self.emote=' {}'.format(self.emoji_warn)
+                self.emote='-_-'
+            elif status['status']=='HEALTH_ERR':
+                self.emote=' {}'.format(self.emoji_err)
+                self.emote='>_<'
+            else:
+                # If statement instead of dict mapping just for this
+                self.emote=' {}'.format(self.emoji_undef)
```

Comment:
also, we don't create a feature PR targeting an old release. especially when this release is EOL. you might want to retarget this change to master.

Reply:
Thx for your review. Yeah I already targeted master, but our cluster runs with nautilus so I thought it would be great if it could be merged. Understandably it can't.

To your points:
1 oh yeah of course
2 oops somehow posted an old release oO
3 I had a dict, but I couldn't use exceptions with this. I feared that status could come up empty or in a future release something different could be added and I didn't want to let my code fail in those circumstances. Error notifications should always work and I didn't want to add unnecessary exception handlers
4 oh yeah sorry bout that

Thx again and have a nice day :)

Reply:
Ok I fixed everything in both branches. #43026 if you are interested :) I will close this pull request here since it's against an EOL version